debug.log

```diff
@@ -1 +1,3 @@
 [1028/120014.369:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
+[1115/122056.566:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1115/124235.273:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
What did you add debug.log file for? I must be ignored by git.

Reply:
Added ***.log** to **.gitignore**. Not sure if it will fix the issue.